job-server/src/main/resources/application.conf

```diff
@@ -124,6 +129,26 @@ spark {
 akka {
   # Use SLF4J/logback for deployed environment logging
   loggers = ["akka.event.slf4j.Slf4jLogger"]
+
+  actor {
+    provider = "akka.cluster.ClusterActorRefProvider"
```

Comment:
You can't know at startup time how many contexts a user will spin up, but there is a flag to turn this off. 

-Evan
"Never doubt that a small group of thoughtful, committed citizens can change the world" - M. Mead

> On Dec 13, 2015, at 12:42 AM, Helena Edelson <notifications@github.com> wrote:
> 
> In job-server/src/main/resources/application.conf:
> 
> > @@ -124,6 +129,26 @@ spark {
> >  akka {
> >    # Use SLF4J/logback for deployed environment logging
> >    loggers = ["akka.event.slf4j.Slf4jLogger"]
> > +
> > +  actor {
> > +    provider = "akka.cluster.ClusterActorRefProvider"
> Always spin up a cluster even for those not needed > 1 context? My one thought is it's a lot of chatter for not needing this new feature, unless leveraging things like leader election etc regardless :)
> 
> —
> Reply to this email directly or view it on GitHub.
>